.github/workflows/test_weaver-fabric-fabric-data-sharing.yaml

```diff
@@ -0,0 +1,1044 @@
+# Copyright IBM Corp. All Rights Reserved.
```

Comment:
This file is for what purpose?

Reply:
In this file I thought it is better to have the test done between two fabric networks only as a starting point.